### ISSUE TITLE
Running in Parallel 

### ISSUE THREAD

**Issue by AntonRand** [jciolek/docker-protractor-headless#6]
Are you able to run your protractor tests in parallel?

I have the following configuration file:

```
exports.config = {
  framework: "jasmine2",
  specs: ['/protractor/tests/**/in*-spec.js'],
  baseUrl: "https://development.myserver.com",
  // Chrome is not allowed to create a SUID sandbox when running inside Docker  
  capabilities: {
    'browserName': 'chrome',
    'shardTestFiles': true,
    'maxInstances': 2,
    'count': 1,
    'chromeOptions': {
      'args': ['no-sandbox']
    }
  },
  'verboseMultiSessions': true
};
```

I find that the first test runs fine but the second one fails with the following:

```
[chrome #01-1] Specs: /protractor/tests/directives/information-modal/information-modal-test-spec.js
[chrome #01-1]
[chrome #01-1] [13:33:41] I/local - Starting selenium standalone server...
[chrome #01-1] /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:654
[chrome #01-1]     throw error;
[chrome #01-1]     ^
[chrome #01-1]
[chrome #01-1] Error: Timed out waiting for the WebDriver server at http://172.17.0.2:35183/wd/hub
[chrome #01-1]     at Error (native)
[chrome #01-1]     at onError (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/http/util.js:87:11)
[chrome #01-1]     at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1379:14)
[chrome #01-1]     at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2913:14)
[chrome #01-1]     at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2896:21)
[chrome #01-1]     at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2775:27)
[chrome #01-1]     at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:639:7
[chrome #01-1]     at process._tickCallback (internal/process/next_tick.js:103:7)
```

**Comment by jciolek** [comment]
In theory you should be able to if you drop `--net=host`, which you can do if you don't run your tests against  localhost.

**Comment by jciolek** [comment]
This issues should be closed by 64d8968
